packages/@aws-cdk/aws-lambda/README.md

```diff
@@ -272,6 +272,32 @@ const fn = new lambda.Function(this, 'MyFunction', {
 See [the AWS documentation](https://docs.aws.amazon.com/lambda/latest/dg/concurrent-executions.html)
 managing concurrency.
 
+### Configuring AutoScaling
+
+You can configure autoscaling of provisioned concurrency on a function alias using the `autoScaleProvisionedConcurrency()` method:
+
+```ts
+const alias = new lambda.Alias(stack, 'Alias', {
+  aliasName: 'prod',
+  version,
+});
+
+// Configure autoscaling to track utilization
+alias.autoScaleProvisionedConcurrency({ minCapacity: 1, maxCapacity: 50 }).scaleOnUtilization({
+  targetUtilizationValue: 0.5,
+});
+
+// or you can configure schedule scaling
+alias.autoScaleProvisionedConcurrency({ minCapacity: 1, maxCapacity: 50}).scaleOnSchedule('ScaleUpInTheMorning', {
+  schedule: appscaling.Schedule.cron({ hour: '8', minute: '0'}),
+  minCapacity: 20,
+});
```

Comment:
I would separate the two operations to emphasize that once you added auto scaling you can configure it with both options:

```ts
const as = alias.addAutoScaling({ maxCapacity: 50 }); // min of "1" feels like a reasonable default
as.scaleOnSchedule(...);
as.scaleOnUtilization(...);
```